ydb/docs/en/core/concepts/datamodel/_includes/table-name-rules.md

```diff
@@ -0,0 +1,32 @@
+## Table naming rules
+
+Every table in {{ ydb-short-name }} has a name. In YQL statements, table names are specified by identifiers that can be enclosed in backticks or not. For more information on identifiers, refer to [{#T}](../../../yql/reference/syntax/lexer.md#keywords-and-ids).
```

Comment:
Let's move all this backticks and identifiers story to the note on line 18. Currently, it is kinda split into two parts, and the reader might notice only half of it and get confused.

Then, only the first sentence would remain in this paragraph, and it can be immediately followed by text from line 5.

Reply:
After discussion, deleted the note and kept the info in the introductory paragraph.